manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass

```

```suggestion
PackageIdentifier: ICEMortgageTechnologyInc.Encompass
```
Commas are not preferred in package identifiers